index.html

```diff
@@ -1336,6 +1339,91 @@ <h2>
 detail how revocation is performed and tracked.
       </p>
 
+      <p>
+All public key formats MUST be expressed in either JSON Web Key (JWK)
+format using the <code>publicKeyJwk</code> property or one of the
+formats listed in the table below. Public key expression MUST NOT use any other key format.
+      </p>
+
+      <p class="issue">
+The Working Group is still debating whether the base encoding format used
+will be Base58 (Bitcoin) [[BASE58]], base64url [[RFC4648]] or base16 (hex) [[RFC4648]]. The entries in the
```

Comment:
Change the base64url reference from RFC4648 to RFC7515, since it specifies no padding

Reply:
@selfissued - Maybe you could bring this up in a new PR/issue, so this one (which was OK with you three days ago) can be merged?

Reply:
I filed this review during the call because I'd raised this in a comment last week and @msporny said on the call that he'd addressed all the comments but apparently mine was missed.  It's a 4-character change - then it can be merged.

Reply:
I see [your previous comment](https://github.com/w3c/did-core/pull/100#pullrequestreview-327053977) now. Took some digging, as github doesn't reveal it when searching for `rfc7515` nor `7515`. I'm putting this in now *as an explicit change request*.

Reply:
Applied in https://github.com/w3c/did-core/pull/100/commits/5287f429d14aefa26d691afd7d868057ffe0e68e.